README.md

```diff
@@ -168,6 +169,44 @@ require("obsidian").setup({
 })
 ```
 
+#### Templates support
+
+To insert a template, run the command `:ObsidianTemplate`. This will open [telescope.nvim](https://github.com/nvim-telescope/telescope.nvim) and allow you to select a template from the templates folder. Select a template and hit `<CR>` to insert. Substitution of `{{date}}`, `{{time}}`, and `{{title}}` is supported. 
+
+For example, with the following configuration
+
+```lua
+require("obsidian").setup({
+  dir = "~/my-vault",
+  templates = {
+      subdir = "my-templates-folder",
+      date_format = "%Y-%m-%d-%a",
+      time_format = "%H:%M"
+  }
+})
+```
+and the file `~/my-vault/my-templates-folder/note template.md`:
+
+```markdown
+---
+aliases = []
+---
+
```

Comment:
By default Obsidian handles the frontmatter by itself, so I think we keep this example consistent with the default settings and omit frontmatter in the template (otherwise you'd end up with duplicate frontmatter).
```suggestion
```

Reply:
Agreed - I overlooked this as I disabled that feature